activity/manager.go

```diff
@@ -67,7 +67,10 @@ func New(idleTimeout, stopRetryPeriod time.Duration) (Manager, error) {
 
 	workspaceName, isFound := os.LookupEnv("CHE_WORKSPACE_NAME")
 	if !isFound {
-		return nil, errors.New("CHE_WORKSPACE_NAME env must be set for activity manager works correctly")
+		workspaceName, isFound = os.LookupEnv("DEVWORKSPACE_NAME")
+		if !isFound {
+			return nil, errors.New("CHE_WORKSPACE_NAME env or DEVWORKSPACE_NAME env must be set for activity manager works correctly")
```

Comment:
```suggestion
			return nil, errors.New("CHE_WORKSPACE_NAME or DEVWORKSPACE_NAME environment variables must be set for activity manager to work correctly")
```

Reply:
thanks @amisevsk good catch! that sounds better :slightly_smiling_face: